themes/clr-installer.theme

```diff
@@ -101,6 +111,8 @@ TableView=─│┼▼▲
 ManualPartition.Back = blue
 ErrorLabel.Back = blue
 ErrorLabel.Text = red bold
+WarningLabel.Back = red
+WarningLabel.Text = white
```

Comment:
Somehow these didn't show up in red background and white text. 
Am i missing something?
![warning](https://user-images.githubusercontent.com/54287843/72840556-5c7a0080-3c49-11ea-815e-6fff16f3644a.png)

Reply:
This screenshot is from the high-contrast TUI? Your comment is on the clr-installer.theme file. There is a high-contrast.theme file below.

Reply:
> Somehow these didn't show up in red background and white text.
> Am i missing something?
> ![warning](https://user-images.githubusercontent.com/54287843/72840556-5c7a0080-3c49-11ea-815e-6fff16f3644a.png)

The high contrast is using the black and white only colors in the `high-contrast.theme`.